Purchases/Purchasing/ProductDetails.swift

```diff
@@ -24,6 +24,14 @@ public typealias SK2Product = StoreKit.Product
 
 /// Abstract class that provides access to all of StoreKit's product type's properties.
 @objc(RCProductDetails) public class ProductDetails: NSObject {
+    public override init() {
+        super.init()
+
+        if self.localizedTitle.isEmpty {
```

Comment:
We can change this to `self.localizedTitle.trimmingWhitespacesAndNewLines.isEmpty` once #995 is merged.